src/renderer/i18n/nl.js

```diff
@@ -0,0 +1,483 @@
+      steps: {
+        factoryRestore: "Terugzetten naar fabrieksinstellingen",
+        bootloaderTrigger: "Bootloader triggeren",
+        bootloaderWait: "Wachten op bootloader",
+        saveEEPROM: "EEPROM inhoud opslaan",
+        restoreEEPROM: "EEPROM inhoud terugzetten",
+        reconnect: "Opnieuw verbinden",
+        flash: "Flashen",
+        reboot: "Opnieuw opstarten",
+      },
+    },
+    confirmDialog: {
+      title: "Firmware vervangen en terugzetten naar fabrieksinstellingen?",
+      contents: `Dit zal de firmware op het toestel vervangen en alle instelling terugzetten naar de fabrieksinstellingen. Je zal alle gemaakte aanpassingen verliezen.`,
+    },
+    defaultFirmware: "Chrysalis {{version}} default",
+    defaultFirmwareDescription: "Minimaal, zonder toeters en bellen",
+    experimentalFirmware: "Chrysalis {{version}} experimenteel",
+    experimentalFirmwareDescription: "Experimenteel, met meer plugins aangezet",
+    selected: "Geselecteerde firmware",
+    custom: "Custom firmware",
+    description: `Updaten of "flashen" van de firmware van je toetsenbord is hoe we het nieuwe kunstjes leren. Chrysalis zal een nieuwe versie van de firmware van je toestenbord installeren. Dit zorgt voor ondersteuning van een aangepaste toetsen layout, naast andere features. Indien je hiervoor al de firmware van je toetsenbord hebt aangepast, zal dit de aangepaste firmware overschrijven. Je kan de broncode van de firmware die Chrysalis aan het installeren is hier vinden:`,
+    postUpload: `Wanneer de upload gedaan is, zal Chrysalis je terugbrengen naar het selectiescherm voor je toetsenbord.`,
+  },
+  "focus-not-detected": {
+    title: "Welkom bij Chrysalis",
+    contents: `Chrysalis herkent je toetsenbord maar moet de firmware updaten voordat je verder kan doen.`,
+    gotoUpdate: "Update Firmware",
+    reconnect: "Opnieuw verbinden",
+    reconnectDescription: `Er is een kans dat we de mogelijkheden van je toetsenbord fout gedetecteerd hebben, of dat je toetsenbord nog aan het opstarten was terwijl we verbinding gemaakt hebben. In dat geval kan je proberen om te klikken op de "{{buttonName}}" knop om opnieuw te proberen verbinden en nogmaals te zoeken naar de nodige features. Opnieuw verbinden is normaal enkel nuttig als je zeker bent dat er een tijdelijk probleem was tijdens vorige pogingen dat in de tussentijd is opgelost.`,
+  },
+  systeminfo: {
+    title: "Rapporteer een probleem",
+    intro: `Als je een probleem hebt met Chrysalis kan het ontwikkelteam je vragen om hen wat debug informatie door te sturen. Chrysalis kan je helpen om alles dat je eventueel moet delen te verzamelen en in te pakken. Dit omvat error logs en informatie over het besturingssysteem, verbonden toestellen en Chrysalis zelf. Toetsenbord informatie omvat info over de huidige firmware, de toets layouts en de LED configuratie van je toetsenbord.`,
+    bugTracker: `Je kan een lijst van gekende problemen vinden op GitHub (of een bug rapporteren)`,
```

Comment:
```suggestion
    bugTracker: `Je kan een lijst van bekende problemen vinden op GitHub (of een bug rapporteren)`,
```

Reply:
https://taaladvies.net/gekend-of-bekend/

I'll change it as apparently "gekend" is less used in this sense outside of Belgium